vhs-frontend/src/App.tsx

```diff
@@ -1,11 +1,11 @@
 import { Routes, Route } from 'react-router-dom';
-import { Header } from './components/Header';
-import { Home } from './pages/Home';
-import { Explore } from './pages/Explore';
-import { VHSDetails } from './pages/VHSDetails';
-import { Edit } from './pages/Edit';
-import { CreateNew } from './pages/CreateNew';
-import { NotFound } from './pages/NotFound';
+import { Header } from '@components';
+import { Home } from '@pages';
```

Comment:
You can put imports from the same place in the same line.

I.e. `import { Home, Explore, VHSDetails, Edit, CreateNew, NotFound } from '@pages';`